packages/cactus-test-geth-ledger/src/main/typescript/geth-test-ledger.ts

```diff
@@ -178,6 +196,7 @@ export class GethTestLedger {
           Env: this.envVars,
           HostConfig: {
             PublishAllPorts: true,
+            networkMode: this.networkName,
```

Comment:
@LordKubaya According to the Docker Engine docs this needs to be capitalized to `NetworkMode`

Source: https://docs.docker.com/reference/api/engine/version/v1.49/#tag/Container/operation/ContainerCreate